testsuite/cluster-test/ct

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# Copyright (c) The Libra Core Contributors
+# SPDX-License-Identifier: Apache-2.0
+set -e
+
+DOCKER_IMAGE="cluster-test:latest"
+TIMEOUT="45m"
+CONTAINER="cluster-test-$RANDOM"
+WORKPLACE="cluster-test"
+
+while (( "$#" )); do
+  case "$1" in
+    -c|--container)
+      CONTAINER=$2
+      shift 2
+      ;;
+    -P|--prune)
+      docker image prune -f
+      shift 1
+      ;;
+    -i|--image)
+      DOCKER_IMAGE=$2
+      shift 2
+      ;;
+    -t|--timeout)
+      TIMEOUT=$2
+      shift 2
+      ;;
+    -w|--workplace)
+      WORKPLACE=$2
+      shift 2
+      ;;
+    --) # end argument parsing
```

Comment:
Do we need a `--pull` flag so that this script can pull the `latest` tag?

Reply:
Hmm.
So if `ct` ran without `--image` it will use image that is built locally, at least this is what I needed for the most time so far.

When Circle runs it, it will build and push docker image to ECR and will pass full image name there (e.g. REPO:UPSTREAM_TAG). It does not require --pull, because it will be unique tag every time